abc.templates/infra/contents/main.tf

```diff
@@ -4,12 +4,11 @@ locals {
 }
 
 module "REPLACE_MODULE_NAME" {
-  source = "git::https://github.com/abcxyz/github-metrics-aggregator.git//terraform?ref=v0.0.14"
+  source = "git::https://github.com/abcxyz/github-metrics-aggregator.git//terraform?ref=v0.0.15"
 
   project_id = local.project_id
 
   image                             = "gcr.io/cloudrun/placeholder@sha256:f1586972ac147796d60ee2c5a0d6cc78067fc862b6d715d6d2a96826455c3423"
-  prefix_name                       = "REPLACE_CUSTOM_NAME"
```

Comment:
Removing because prefix has 17 character limit due to wif service account name limitation of 28 characters.
It's unlikely that there will be multiple GMA instances, so I think we can just accept the default prefix.